quickstart/quickstart.ipynb

```diff
@@ -21,17 +25,36 @@
    "outputs": [],
    "source": [
     "import coiled\n",
+    "\n",
+    "cluster = coiled.Cluster(n_workers=10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once a cluster has been created (you can see the status on your [coiled dashboard](https://cloud.coiled.io/)), you can connect Dask to the cluster by creating a `distributed.Client` instance."
```

Comment:
"coiled dashboard" -> "Coiled dashboard"

Here and a few other places in this notebook.